pkg/config/fetcher/from_processes.go

```diff
@@ -27,3 +28,28 @@ func FetchSecurityAgentConfig(config config.Reader) (string, error) {
 	client := settingshttp.NewClient(c, apiConfigURL, "security-agent")
 	return client.FullConfig()
 }
+
+// ProcessAgentConfig fetch the configuration from the process-agent process by querying its HTTPS API
+func ProcessAgentConfig(config config.Reader) (string, error) {
```

Comment:
I think `ProcessAgentConfig` and `SecurityAgentConfig` could be factored in a single function with some parameters (in particular if there'll be the same for the remaining processes)